_sass/components/_navbar.scss

```diff
@@ -1,6 +1,6 @@
 // Styling for the navbar
 #mainNav {
-  background-color: $gray-900;
+  background-color: $medium-gray!important;
```

Comment:
think you're missing a space here? The rest look good!

Reply:
It'll work without one, but it looks more readable with one so I just added it in